ZelBack/src/lib/socket.js

```diff
@@ -40,6 +40,7 @@ function initIO(httpServer) {
         Tty: true,
         Cmd: serviceHelper.commandStringToArray(dockerCmd),
         Env: serviceHelper.commandStringToArray(dockerEnv),
+        User: dockerUser,
```

Comment:
what if this is undefined? as of different fluxos versions

Reply:
if it is not set is used def username defined in dockerfile or root